### PR TITLE
Fix [RTL]: mirror new user tour modal background

### PR DESCRIPTION
**Pull request by ARMBouhali**
**TL;DR -** When in RTL, it mirrors the background image used in the new user tour modal as it obstructs the readability of the modal title.

**Screenshots**

| LTR for reference | Screenshot from 2022-10-03 13-20-57 |
| ---- | ---- |
| RTL before fix | Screenshot from 2022-10-03 13-22-47 |
| RTL after fix | Screenshot from 2022-10-03 13-21-27 |

**Testing instructions**

- It should be your first time as a user to access a course home, otherwise either create and enroll a new user or edit your account's UserTour object from LMS Django admin.
- Switch to your user language preference to an RTL language of your choice (e.g. Arabic).
- Go to your course home. The "Welcome to your platform course" modal should appear.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ARMBouhali!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @ARMBouhali - is this ready for review?

**Comment by ARMBouhali** [comment]
> Hi @ARMBouhali - is this ready for review?
Yes It I's ready

**Comment by mphilbrick211** [comment]
@mattcarter could you please enable the tests to run?

**Comment by ARMBouhali** [comment]
I've modified the PR code to become pure CSS + 1 className.
Waiting for reviewers

**Comment by mphilbrick211** [comment]
@brian-smith-tcril - would you mind taking a look at this? It's ready for review.

**Review by brian-smith-tcril — APPROVED** [review]
LGTM! :shipit:

**Review by arbrandes — APPROVED** [review]
+1

**Comment by openedx-webhooks** [comment]
@ARMBouhali 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-learning&entry.867055334=https://github.com/openedx/frontend-app-learning/pull/985&entry.1484655318=https://github.com/ARMBouhali&entry.752974735=2022-10-03+12:43&entry.1917517419=2022-11-07+16:04&entry.2133058324=Yes) so we can improve your experience in the future.
